JSIL/AST/JSAstVisitor.cs

```diff
@@ -216,6 +233,11 @@ protected class Adapter<TVisitor, TNode>
                 NodeIndex = oldNodeIndex;
                 StatementIndex = oldStatementIndex;
             }
+
+            if (node != null && AfterNodeProcessed != null)
```

Comment:
I've seen it when I translate my application. I've a lot of log messages, but as it doesn't affect anything I've not looked on it.